specs/dwritecore/DWriteCore.md

```diff
@@ -259,5 +1127,87 @@ DWRITE_BEGIN_INTERFACE(IDWriteBitmapRenderTarget2, "C553A742-FC01-44DA-A66E-B8B9
     /// </summary>
     STDMETHOD(GetBitmapData)(_Out_ DWRITE_BITMAP_DATA_BGRA32* bitmapData) PURE;
 };
-#endif // DWRITE_CORE
+```
+
+## IDWriteFontSet4 Interface
+
+```c++
+DWRITE_BEGIN_INTERFACE(IDWriteFontSet4, "EEC175FC-BEA9-4C86-8B53-CCBDD7DF0C82") : IDWriteFontSet3
+{
+    /// <summary>
+    /// Computes derived font axis values from the specified font weight, stretch, style, and size.
+    /// </summary>
+    /// <param name="inputAxisValues">Pointer to an optional array of input axis values. Any axes present
+    /// in this array are not included in the output. This is so explicit axis values take precedence over 
+    /// derived axis values.</param>
+    /// <param name="inputAxisCount">Size of the array of input axis values.</param>
+    /// <param name="fontWeight">Font weight, used to compute "wght" axis value.</param>
+    /// <param name="fontStretch">Font stretch, used to compute "wdth" axis value.</param>
+    /// <param name="fontStyle">Font style, used to compute "slnt" and "ital" axis values.</param>
+    /// <param name="fontSize">Font size in DIPs, used to compute "opsz" axis value. If this parameter is zero,
+    /// no "opsz" axis value is added to the output array.</param>
+    /// <param name="outputAxisValues">Pointer to an output array to which derived axis values are written.
+    /// The size of this array must be at least DWRITE_STANDARD_FONT_AXIS_COUNT (5). The return value is 
+    /// the actual number of axis values written to this array.</param>
```

Comment:
DISCUSSION: This is just a place to expose the method. The GetDerivedFontAxisValues is the conversion tool from the old value types to the new axis types - this method fills in the gaps of what you don't yet have. If the input set contains weight, the output set will _not_ contain weight, as it's not derived - you know it upfront.

SAMPLE: Revisit Raymond's "here's how to combine everything in a single call" model, optionally.

Reply:
Just clarifying that 5 elements is sufficient. The existing docs said that it must be "at least 5", which opened the question of "are there cases where it might write 6"? (Answer: No. It will never write 6.)